islandora.api.php

```diff
@@ -614,7 +614,7 @@ function hook_cmodel_pid_islandora_overview_object(AbstractObject $object) {
  * Lets one alter the overview tab in object management.
  */
 function hook_islandora_overview_object_alter(AbstractObject &$object, &$output) {
-  $output = $output . drupal_render(drupal_get_form('some_form', $object));
+  $output = $output . drupal_render(drupal_get_form('some_form', $object)));
```

Comment:
I know this is just example stuff, but someone attempting to use this would get notices... `drupal_render()` takes a reference, which means that you need to stash the value into a variable before passing it along... Something like:
```php
$form = drupal_get_form('that_thing', $object);
$output = $output . drupal_render($form);
```